### DISCUSSION TITLE
GitHub debería tener la opción de deshabilitar la sugerencia de "Compare & Pull Request"

### DISCUSSION THREAD

**Discussion by SCodify** [community/community#116575] [Discussions]
### Select Topic Area

General

### Body

Supuestamente esta es una característica de GitHub para facilitar la creación de Pull Requests después de hacer un push a una rama. Actualmente, GitHub no ofrece una opción para desactivar estas sugerencias en la interfaz de usuario. Estas sugerencias se muestran a todos los usuarios que tienen permisos de escritura en el repositorio cuando hacen un push a una rama.
Si bien no se pueden desactivar estos mensajes, uno puede optar por ignorarlos si no desea crear un Pull Request en ese momento. Yo creo que poder desactivar esas sugerencias debería ser una opción porque a veces resulta molesto y suelen acumularse los mensajes de "Compare & Pull Request".

**Comment by SCodify** [comment]
Otra contra es que si uno toca ese botón te quiere hacer el merge directamente a la rama main.

**Comment by queenofcorgis** [comment]
Hi @SCodify , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.

**Comment by SCodify** [comment]
![Captura de pantalla 2024-04-02 164938](https://github.com/community/community/assets/104208569/47e9d2df-18a1-423d-90aa-d4e96e91d26d)

**Comment by queenofcorgis** [comment]
Please note:  You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.
